client/signup/steps/theme-selection/index.jsx

```diff
@@ -58,14 +92,38 @@ module.exports = React.createClass( {
 		);
 	},
 
+	renderThemePreview() {
+		if ( this.state.previewTheme ) {
+			return (
+				<ThemePreview
+					showPreview={ true }
```

Comment:
An alternative technique is to always render <ThemePreview/> and toggle the `showPreview` prop to hide/show. This wound allow the transition animations to work.

(The reason we don't do that in the theme showcases is because it interferes with infinite scroll).